combat-trainer.lic

```diff
@@ -1175,7 +1175,17 @@ class TrainerProcess
     when 'App Careful'
       appraise(game_state, 'careful')
     when 'Tactics'
```

Comment:
As @izawwlgood pointed out, some people might want to continue using weave/bob/circle. I think I'd prefer to see a new training ability added instead of replacing Tactics. Maybe `Tactics Analyze`?

That ensures that existing behavior continues for players, but then gives them the option to change if they want.